examples/exit/exit.sh

```diff
@@ -1,13 +1,14 @@
-#  If you run `exit.go` using `go run`, the exit
-# will be picked up by `go` and printed.
+# Se esegui `exit.go` usando `go run`, il valore di uscita
```

Comment:
A dirla tutta qua intendo proprio il valore di uscita come ho trovato su [Wikipedia](https://it.wikipedia.org/wiki/Valore_di_uscita), non il valore di ritorno di una funzione :sweat_smile: